### PR TITLE
Add note for DoubleTree

### LINKED ISSUE #68: Clarify hotel reservations due 0024-07-25
**Request type**: Website change / addition

**Email** christina.maimone@northwestern.edu

**Purpose** Clarify hotel reservations

**Content**  https://docs.google.com/document/d/116aK8MGn9OuF67uHtP59lrpFjeM73M0oxK7AF4W11jQ/edit?usp=sharing 

**Request Drafts** [Google Drive - Request Drafts](https://drive.google.com/drive/folders/16NgJUc6ed3fOWxZssmPrubhf2EdScDC6?usp=share_link)

- [ ] I've informed the requestor that this is complete

### PR DESCRIPTION
**Pull request by jsubida**
## Description

Closes #68 

## Checklist:

- [x] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc **ADD GITHUB HANDLES HERE**

Screenshot 2024-07-23 at 09 09 24

### CODE REVIEW

File: pages/attend/travel.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ US-RSE'24.
 - _Address_: 201 Marquette Avenue NW. Albuquerque, New Mexico 87102 (0.1 miles away)
 - _Phone_: 505-247-3344
 - _Special Rate Group Code_: 90F
+  - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to call the hotel to either make your reservation, or to add the night of the 17th onto a reservation you first create online.  If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.
```
Comment (mrmundt) [inline pages/attend/travel.md:27]:
```suggestion
  - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to either:
  - call the hotel to either make your reservation
  - add the night of the 17th onto an existing reservation that was created online
If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.
```

Comment (mrmundt) [reply pages/attend/travel.md:27]: Just to make it a bit more explicit.

Comment (mrmundt) [reply pages/attend/travel.md:27]: (That may render poorly... _Use caution_)

File: pages/attend/travel.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ US-RSE'24.
 - _Address_: 201 Marquette Avenue NW. Albuquerque, New Mexico 87102 (0.1 miles away)
 - _Phone_: 505-247-3344
 - _Special Rate Group Code_: 90F
+  - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to either:
+  - call the hotel to either make your reservation
+  - add the night of the 17th onto an existing reservation that was created online
+If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.
```
Comment (mrmundt) [inline pages/attend/travel.md:30]:
```suggestion
  - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to either:
     - call the hotel to either make your reservation
     - add the night of the 17th onto an existing reservation that was created online
  - If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.
```

Comment (mrmundt) [reply pages/attend/travel.md:30]: Knew it... Looks weird. Sorry :-/

Comment (jsubida) [reply pages/attend/travel.md:30]: All good! Thanks for feexing

**Review by mrmundt — APPROVED** [review]

**Pull request merged**